examples/counter/deployments/test.yaml

```diff
@@ -0,0 +1,50 @@
+---
+id: 0
```

Comment:
What's the purpose of `id`?
Can we have more than one deployment plan in a single file?

Reply:
You could yes.